engine/src/main/java/org/terasology/physics/bullet/BulletPhysics.java

```diff
@@ -324,7 +324,7 @@ public boolean updateRigidBody(EntityRef entity) {
         RigidBodyComponent rb = entity.getComponent(RigidBodyComponent.class);
         BulletRigidBody rigidBody = entityRigidBodies.get(entity);
 
-        if (location == null || Float.isNaN(location.getWorldPosition().x)) {
```

Comment:
We don't have the precondition check yet - do you think it's safe to remove the finite check here?

Reply:
It doesn't actually use the worldPosition the actual position update happens in EntityMotionState.